flocx_market/tests/unit/objects/test_object_bid.py

```diff
@@ -0,0 +1,64 @@
+from flocx_market.objects import bid
+import datetime
+now = datetime.datetime.utcnow()
+
+
+test_bid_1 = dict(creator_bid_id="1259a51-b4d6-497d-9f75-f56c409305c8",
+                  creator_id="12a59a51",
+                  server_quantity=2,
+                  start_time=now,
+                  end_time=now,
+                  duration=16400,
+                  status="available",
+                  server_config_query={'foo': 'bar'},
+                  cost=11.2)
+
+test_bid_2 = dict(creator_bid_id="12a59a51-b4d6-497d-9f75-f56c409305c8",
+                  creator_id="12a9a51",
+                  server_quantity=2,
+                  start_time=now,
+                  end_time=now,
+                  duration=16400,
+                  status="available",
+                  server_config_query={'foo': 'bar'},
+                  cost=11.5)
+
+
+def test_create(app, db, session):
+    ret = bid.Bid.create(test_bid_1)
+    assert getattr(ret, 'creator_id') == '12a59a51'
```

Comment:
These tests can actually be much simpler, since we don't actually want to insert data into a database; that's for the database unit tests.

Something like:

```
import unittest.mock as mock


@mock.patch('flocx_market.db.sqlalchemy.api.bid_create')
def test_create(bid_create):
    ret = bid.Bid.create(test_bid_1)
    bid_create.assert_called_once()
```